apps/src/codebridge/FileTabs/FileTab.tsx

```diff
@@ -27,14 +27,17 @@ const FileTab = ({file}: FileTabProps) => {
 
   return (
     <div className={className} key={file.id}>
-      <span onClick={() => setActiveFile(file.id)}>
+      <div
+        className={moduleStyles.label}
+        onClick={() => setActiveFile(file.id)}
+      >
         <FontAwesomeV6Icon
           iconName={iconName}
           iconStyle={iconStyle}
           className={iconClassName}
         />
         <span>{file.name}</span>
-      </span>
+      </div>
```

Comment:
Does this work because the div fills the whole horizontal space of the container while the (outer) span was just wide enough to contain the filename text (inner span)?

Reply:
Yeah I also changed the width to 100% to make it fit.

Reply:
span to div wasn't strictly necessary (I think I changed it thinking it might fix it on its own), but I kept it as a div because we generally use divs

Reply:
Sounds like we followed similar trains of thought then! Cool, thanks for the context.